code/modules/chemistry/Reagents-Misc.dm

```diff
@@ -844,28 +821,33 @@ datum
 			var/visible = 1
 
 			reaction_turf(var/turf/target, var/volume)
-				var/visible = src.visible
-				var/turf/simulated/T = target
-				if (istype(T))
-					if (T.wet >= 3) return
-					if (visible)
-						var/image/wet = image('icons/effects/water.dmi',"wet_floor")
-						wet.blend_mode = BLEND_ADD
-						wet.alpha = 60
-						T.UpdateOverlays(wet, "wet_overlay")
-						playsound(T, 'sound/impact_sounds/Slimy_Splat_1.ogg', 50, TRUE)
-					T.wet = 3
-					SPAWN(80 SECONDS)
-						if (istype(T))
-							T.wet = 0
-							T.UpdateOverlays(null, "wet_overlay")
-				return
+				if (istype(target, /turf/simulated))
+					var/turf/simulated/simulated_target = target
+					simulated_target.wetify(3, 60, 60 SECONDS)
```

Comment:
Needs to take into account `visible` to make the traitor sign lube invisible and silent.